### PR TITLE
First commit of a file for Copenhagen

### LINKED ISSUE #87: Create finoutlook hometown file
## Copenhagen, Denmark
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [ ] Have a conversation
- [ ] Get a :+1:
- [ ] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by finoutlook**
This PR resolves #87

### CODE REVIEW

File: finoutlook-copenhagen.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Copenhagen, Denmark
+
+### Places to each
```
Comment (finoutlook) [inline finoutlook-copenhagen.md:3]: This should say 'eat'

Comment (abcfantasy) [inline finoutlook-copenhagen.md:3]: I agree!

**Comment by finoutlook** [comment]
This looks :+1:

**Comment by finoutlook** [comment]
:ship:

**Pull request merged**